Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:20-alpine
+
+RUN mkdir -p /app
+
+COPY . /app
+
+WORKDIR /app
+
+RUN npm i && npm i -g nodemon
```

Why do you need to install `nodemon` separately?
```suggestion
RUN npm i
```

---

Sorry, forgot to remove this